fuzzing/oss-fuzz-scripts/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
This `build.sh` script in particular actually is only intended to be sourced from another script. Specifically, it should only ever be sourced by the file of the same name inside the OSS-Fuzz docker container ([see the corresponding downstream PR for where it's used](https://github.com/google/oss-fuzz/pull/11803/files#diff-f54f040cc212bb97ac727c4a7d617bf604aaa65a442b972d32e0fa3061799378)) that requires the environment specific build steps it performs.

The env vars and `compile_python_fuzzer` function that it depends on are provided by OSS-Fuzz. (That said, some checks for those and a user friendly error if executed without them couldn't hurt I suppose.)


Does that change your perspective on this specific file?

Reply:
> Does that change your perspective on this specific file?

Yes, absolutely, and I apologize for having made a recommendation based on my own inaccurate understanding here.

Since this is meant to be sourced rather than executed, it should not have an executable bit added. But the shebang is, for the same reason, misleading. To continue informing [ShellCheck](https://github.com/gitpython-developers/GitPython/blob/174faa94a8e1545299729d95f3440d9079bdedbb/.pre-commit-config.yaml#L11-L16) that this is Bash code, while avoiding having a `#!` line on a file that is not, even in principle, meant for execution, it can be [changed to](https://www.shellcheck.net/wiki/Directive#shell):

```suggestion
# shellcheck shell=bash
```

Reply:
> Yes, absolutely, and I apologize for having made a recommendation based on my own inaccurate understanding here.

No apologies necessary. I appreciate the insight 🙂 

> Since this is meant to be sourced rather than executed, it should not have an executable bit added. But the shebang is, for the same reason, misleading. To continue informing [ShellCheck](https://github.com/gitpython-developers/GitPython/blob/174faa94a8e1545299729d95f3440d9079bdedbb/.pre-commit-config.yaml#L11-L16) (and human readers) that this is Bash code, while avoiding having a `#!` line in a file that is not, even in principle, meant for execution, it can be [changed to](https://www.shellcheck.net/wiki/Directive#shell):
> 
> ```suggestion
> # shellcheck shell=bash
> ```

Perfect, I'll do that. Thanks!